wrappers/src/sync_session_cs.cpp

```diff
@@ -26,6 +26,8 @@
 #include <realm/object-store/sync/sync_session.hpp>
 #include "sync_session_cs.hpp"
 
+enum class NotifiableProperties : uint8_t;
```

Comment:
Why is this defined here but implemented further down? Can't we just move the implementation here?

Reply:
It was just to keep it a little tidier. If stylistically it's preferred to have it in there, then I'll move the implementation there and remove this forward declaration.

Reply:
I don't know - @fealebenpae you're the c++ master here. I just find it easier to have things defined once, but not going to pretend I know what I'm talking about 😅